ironfish/src/mining/director.ts

```diff
@@ -147,6 +150,10 @@ export class MiningDirector<
         this.logger.error(err)
       })
     })
+
+    if (options.graffiti) {
+      this.setBlockGraffiti(options.graffiti)
+    }
```

Comment:
should we do this with minerAccount as well?

Reply:
lets do it... I actually just did that in the PR that removes captain from the mining director

Reply:
done

Reply:
fixed